notes_app/client/app/login/login.component.js

```diff
@@ -15,7 +15,7 @@
       });
     };
     ctrl.success = function() {
-      ctrl.close();
+      ctrl.close({'$value': true});
```

Comment:
It's a leftover from when the user state was stored in a main component. Now that the user state is stored in a service, it's no longer required and can be removed if necessary.